README.md

```diff
@@ -24,25 +29,27 @@ sudo apt-get install -y zsh git vim
 sudo chsh # change shell to /usr/bin/zsh and restart
 sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
 git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
+git clone https://github.com/zsh-users/zsh-autosuggestions ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autosuggestions
+
+cat > $HOME/.zshrc << END
```

Comment:
I'm not sure it's really the complete `.zshrc` in the end, we should check with Yotam (before he gives the laptop back)

Reply:
I just made it more script-able since I was running it a lot